rhythm/constantdelay.go

```diff
@@ -20,13 +20,37 @@ func Every(duration time.Duration) ConstantDelaySchedule {
 	if duration < time.Second {
 		duration = time.Second
 	}
+
 	return ConstantDelaySchedule{
 		Delay: duration - time.Duration(duration.Nanoseconds())%time.Second,
 	}
 }
 
 // Next returns the next time this should be run.
 // This rounds so that the next activation time will be on the second.
-func (schedule ConstantDelaySchedule) Next(t time.Time) time.Time {
-	return t.Add(schedule.Delay - time.Duration(t.Nanosecond())*time.Nanosecond)
+func (schedule ConstantDelaySchedule) Next(start, t time.Time) time.Time {
+	if start.IsZero() {
+		// schedule is not bound to a starting point
+		return t.Add(schedule.Delay - time.Duration(t.Nanosecond())*time.Nanosecond)
```

Comment:
`time.Duration(t.Nanosecond())*time.Nanosecond` What does this achieve? Isn't `t.Nanosecond` fine?

```suggestion
		return t.Add(schedule.Delay.Sub(t.Nanosecond()))
```

Reply:
This is the original code. I wanted to avoid changing it. At this point, I can rewrite this too. We just need to add after truncating to seconds.

Reply:
Done.